docs/src/s2e-env.rst

```diff
@@ -32,7 +32,8 @@ Installing s2e-env
     cd s2e-env
 
     python3 -m venv venv
-    . venv/bin/activate
+    chmod +x ./venv/bin/activate 
+    ./venv/bin/activate
```

Comment:
The command was correct. This script must be sourced.
https://docs.python.org/3/tutorial/venv.html

Reply:
Thanks V. Using `source` is actually what I have done. I'm consider to change it to `source` just right now